src/components/schemas/Prospect.yaml

```diff
@@ -40,3 +40,10 @@ properties:
   savedSearchIds:
     type: number
     format: uuid
+  source:
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->

<!--
Argument | Meaning
-- | --
(non-blocking) |	A comment with this decoration should not prevent the subject under review from being accepted. This is helpful for organizations that consider comments blocking by default.
(blocking)	| A comment with this decoration should prevent the subject under review from being accepted, until it is resolved. This is helpful for organizations that consider comments non-blocking by default.
(if-minor)	| This decoration gives some freedom to the author that they should resolve the comment only if the changes ends up being minor or trivial.
(unit-test)	| Comment with this decoration suggests something related to unit tests.
(performance)	| Comment with this decoration suggests something related to performance.
-->

**:sparkles:  suggestion (non-blocking):** tu dois pouvoir faire références à `/components/parameters/prospectSource.yaml`

Reply:
Non ce n'est pas possible car `prospectSource` correspond à la description d'un paramètre et non d'un schéma. Il y a ceci dans le fichier yaml `in: query`. On ne peut l'utiliser que dans une description de paramètres.

Par contre si on compare avec le fichier `/components/schemas/Property.yaml`, on voit que certaines enum sont définies en dur comme la mienne ou d'autres comme la source d'un bien sont définies dans un autre schéma comme `/components/schemas/Property-Source.yaml`. Je pense que la source d'un bien est utilisé par d'autres objets d'où le fichier à part.

Dans le cas du prospect, la source n'est pas utilisée ailleurs pour le moment. Mais je peux l'externaliser dans un autre schéma en prévision.

Que préfères-tu ?